go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/samuelattwood/partner-charts-ci
+module github.com/rancher/partner-charts-ci
```

Comment:
@adamkpickering This should have been updated through https://github.com/rancher/partner-charts-ci/pull/2, correct?

Reply:
probably yes I think

Reply:
Yes, it was. This PR just needed to be rebased to use those changes.

Reply:
@nicholasSUSE How did you create this branch?